### DISCUSSION TITLE
Can't start update

### DISCUSSION THREAD

**Discussion by fshubthunfischbaum** [mediacms-io/mediacms#855] [Q&A]
**Describe the issue**
Cannot pull git source

**To Reproduce**
Steps to reproduce the issue:
1. "source activate"
2. Type "git pull"
3. receive this error "fatal: not a git repository (or any of the parent directories): .git"

**Expected behaviour**
pull should succeed

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04.6 LTS (GNU/Linux 5.4.0-153-generic x86_64)
 - Installation method: Single Server

**Additional context**
I'm probably missing something tremendously on my end here. But somehow cannot figure it out. Bump in the right direction would be very much appreciated.

**Comment by mgogoulos** [comment]
Hi, are you in the correct directory? Most probably not, so make sure you get into `/home/mediacms.io/mediacms/`

Then check that this is actually is the correct directory with `git remote -v` , you should see something like this:
```
root@server /home/mediacms.io/mediacms # git remote -v
origin	https://github.com/mediacms-io/mediacms (fetch)
```

**Reply by mgogoulos** [reply]
did you try this out @fshubthunfischbaum ?

**Comment by Dabitch** [comment]
he might have to add 

git config --global --add safe.directory /home/mediacms.io/mediacms 

before  trying 
git remote -v
